### ISSUE TITLE
Feature Request: Hide peek line or disable peek

### ISSUE THREAD

**Issue by shivangswain** [White-Tiger/T-Clock#209]
Something like this would look super clean:

![image](https://user-images.githubusercontent.com/33395916/87642263-e91ffa00-c766-11ea-9bc9-cff6b5edb178.png)

![image](https://user-images.githubusercontent.com/33395916/87642286-efae7180-c766-11ea-91ff-ea946e0d53ad.png)

**Comment by White-Tiger** [comment]
Are you talking about the "show desktop" button on the far right?
![image](https://user-images.githubusercontent.com/1467733/87642980-8f113b80-c74a-11ea-9362-7a266a9d3d4f.png)

**Comment by shivangswain** [comment]
Yes, that is exactly what I am talking about. I have currently removed it using this tool for Windows 7 called [ShowDesktop Remover](https://www.thewindowsclub.com/remove-and-restore-show-desktop-button-with-showdesktop-remover) but I would love if it was available included in T-Clock.

**Comment by White-Tiger** [comment]
It's a simple tool that only hides the button (doesn't clear the space), so I wouldn't mind if T-Clock did the same.

**Comment by LumKitty** [comment]
Take a look at: https://rammichael.com/7-taskbar-tweaker which can get rid of the show desktop button (freeing up the space), and many other things.

I currently use it in conjunction with T-Clock Redux and WinAero tweaker to get my taskbar looking like this:

![image](https://user-images.githubusercontent.com/8736448/87811283-f1069980-c84d-11ea-97ac-5059f71af926.png)

**Comment by shivangswain** [comment]
@LumKitty I have used 7++ Taskbar Tweaker in the past, even donated to get access to beta versions. But you cannot use it for every version of Windows 10. They release a version for every Windows release and if you are using a newer version or an Insider Build, they lock you out of the application. That is why I went with ShowDesktop Remover.

Edit: Would you mind showing us your whole taskbar setup? Just curious.

**Comment by LumKitty** [comment]
eh? the rest isn't anything special...

Also I think 7TT does now let you attempt to run it against a newer build, though it's rarely an issue for me as I usually leave it a month or so before updating to a new Win10 version. Let other people find the bugs!

![image](https://user-images.githubusercontent.com/8736448/87852934-b22c1e80-c8f5-11ea-9348-c74ae129e334.png)

**Comment by Schattenspieler** [comment]
### @shivangswain regarding 7TT:
> But you cannot use it for every version of Windows 10. They release a version for every Windows release and if you are using a newer version or an Insider Build, they lock you out of the application.

It just gives you a warning and the option to proceed anyway, if an unknown version is detected.
But if I'm not mistaken, that changed over the last ~2 years or so.

Ah, to [quote the author](https://rammichael.com/7-taskbar-tweaker-and-windows-10-insider-builds) (2018-08-06):
> Just like previously with alpha versions, support for insider builds will be only available to those who supported the project with a donation.
> ...
> Starting from 7+ Taskbar Tweaker v5.5.0.3 (beta), experimental support for new Windows builds will be released with public beta versions, but will be locked with an access code. The access code will be available to those who decide to support the project, just like it was previously with alpha versions.
